src/lvl_script.c

```diff
@@ -239,6 +243,7 @@ const struct NamedCommand player_desc[] = {
   {"PLAYER_GOOD",      PLAYER_GOOD},
   {"ALL_PLAYERS",      ALL_PLAYERS},
   {"PLAYER_NEUTRAL",   PLAYER_NEUTRAL},
+  {"ANY_KEEPER",       6},
```

Comment:
Could you elaborate in which situations you feel this variable could be useful?

Reply:
If you want to "booby trap" a level with hero party spawns, but of course don't want the heroes themselves to trigger it,

Reply:
Do heroes trigger action points now?

Reply:
I'm pretty sure they did when I tested ALL_PLAYERS.

Reply:
You're probably right,... though I'm pretty unhappy with the situation. ALL_PLAYERS really is all players in most cases, but for action points it's 'any player'. I get that's a much more common use case, but still. Changing the behavior for ALL_PLAYERS on action points is really not an option since that would probably break a lot of old maps.

Having a ANY_KEEPERS command really highlights the difference for ALL_PLAYERS though, and I feel it just adds to the confusion. I think we should just forget about this one, and when people do want to trigger on multiple players, have them script it out. It's extremely rare - perhaps non-existant - that maps with many keepers reach the IF-limit, and this behavior could easily be scripted out.

For all other cases were players are allowed as a parameter, I don't think this variable could have any meaning either.